keras_cv/models/darknet.py

```diff
@@ -44,11 +44,11 @@
         https://keras.io/guides/transfer_learning/).
 
     Args:
-        include_rescaling: whether or not to Rescale the inputs.If set to True,
+        include_rescaling: bool, whether or not to Rescale the inputs.If set to True,
```

Comment:
I'm not sure these added types clarify the docstrings.  Can we revert?

Reply:
@LukeWood -- @jbischof added types to these docstrings in ResNet when he did the proof of concept PR for this approach, and we've been including them everywhere else for consistency.